### PR TITLE
Revert "Adding oc_process to lib_openshift."

### PR DESCRIPTION
**Pull request by ingvagabund**
Reverts openshift/openshift-ansible#3250

### CODE REVIEW

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
Strange, the failures are not related to the PR. Maybe something in the CI itself.

File: roles/lib_openshift/tasks/main.yml [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
-    name: "{{ item }}"
+    name: python-ruamel-yaml
     state: present
-  with_items:
-  - ruamel.yaml
-  - ruamel.yaml
-  - ruamel.yaml
-  - ruamel.yaml
-  - ruamel.yaml
-  - ruamel.yaml
```
Comment (sdodson) [inline roles/lib_openshift/tasks/main.yml]: Any reason not to reduce this PR to just this change?

Comment (sdodson) [inline roles/lib_openshift/tasks/main.yml]: Also, lol at having it in there 6 times.

**Comment by sdodson** [comment]
As in don't revert the inclusion of oc_process, just fix the incorrect package installation.

**Comment by ingvagabund** [comment]
Ok, given `python-ruamel-yaml` is already used in other modules, I will just fix the one affected file.

**Comment by ingvagabund** [comment]
Closing in favor of https://github.com/openshift/openshift-ansible/pull/3327

**Pull request closed without merging**